src/main/java/org/example/ngevaticketmanagerspring/user/UserService.java

```diff
@@ -0,0 +1,63 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.springframework.stereotype.Service;
+
```

Comment:
```suggestion

@RequiredArgsConstructor
```

Wie im Controller, um sich das Schreiben des Konstruktors zu sparen.